### PR TITLE
Stopped all animation controllers after toggleable has been detached.

### PR DESCRIPTION
**Pull request by MariaMelnik**
## Description

Added stop animation controllers in detach.
Without it exception will be produced in case object is removed from the tree quicker than animation finished.

### CODE REVIEW

**Comment by MariaMelnik** [comment]
Closed as AUTHORS wasn't edited.

**Comment by MariaMelnik** [comment]
AUTHORS edited correctly.

File: examples/hello_world/ios/Flutter/.last_build_id [hunk 1]
```diff
@@ -1 +1 @@
-8815a07f6ac6b55e6bd0d2350525ef75
\ No newline at end of file
+ca8d5726b688e70f8dd95aad372bf422
```
[the remark this answers is not in the document]
Comment (MariaMelnik) [reply examples/hello_world/ios/Flutter/.last_build_id]: fixed (removed this from pr)

File: packages/flutter/test/material/checkbox_test.dart [hunk 2]
```diff
@@ -603,6 +603,56 @@ void main() {
...
+            child: Center(
+              child: Container(),
+            ),
+          ),
+        ),
+      ),
+    );
+
+    await testGesture.removePointer();
+  }, variant: const TargetPlatformVariant(<TargetPlatform>{TargetPlatform.macOS}));
```
Comment (shihaohong) [inline packages/flutter/test/material/checkbox_test.dart:653]: Is there a reason we're only testing this on macOS?

Comment (MariaMelnik) [reply packages/flutter/test/material/checkbox_test.dart]: @shihaohong yes, this is redundant. Removed it.

File: packages/flutter/test/material/checkbox_test.dart [hunk 3]
```diff
@@ -603,6 +604,56 @@ void main() {
...
+          child: Material(
+            child: Center(
+              child: Container(),
+            ),
+          ),
+        ),
+      ),
+    );
+
+    await testGesture.removePointer();
```
Comment (shihaohong) [inline packages/flutter/test/material/checkbox_test.dart:652]: Add 
```dart
      // Hover animation should not trigger an exception when the checkbox is removed
      // before the hover animation should complete.
      expect(tester.takeException(), isNull);
```

This signifies to future readers or contributors that the point of this test is that an exception does not occur in the above situation.

Comment (MariaMelnik) [reply packages/flutter/test/material/checkbox_test.dart:656]: @shihaohong added.

**Review by shihaohong — APPROVED** [review]
LGTM

**Pull request merged**